NASA-AMMOS/aerie-ui
# Delete Anchored Directives UI

resolves #397 

To test:
1. Create a view with several directives, some of which are anchored to others
2. Select a few directives (be sure to include at least one that has a directive anchored to it)
3. Ensure that deleting is successful for all three options presented
4. Ensure that deleting a directive with nothing anchored to it still works
